src/components/ButtonToggle/ButtonToggle.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {StoryObj, Meta, StoryContext} from '@storybook/react';
+
+import {ButtonToggle} from './index';
+import type {ButtonToggleProps} from './ButtonToggle.types';
+
+import {Apps} from '~/icons';
+import markdownNotes from './ButtonToggle.md';
+
+const meta: Meta<typeof ButtonToggle> = {
+    title: 'Components/ButtonToggle',
+    component: ButtonToggle,
+
+    parameters: {
+        layout: 'centered',
+        actions: {argTypesRegex: '^on.*'},
+        notes: {markdown: markdownNotes}
+    }
+};
+export default meta;
+
+type Story = StoryObj<ButtonToggleProps>;
+const Template = (args: ButtonToggleProps, globals: StoryContext) => {
+    const theme = globals.theme;
+    return <ButtonToggle {...args} isReversed={theme === 'dark'}/>;
+};
+
+export const Default: Story = {
+    args: {
+        size: 'default',
+        iconStart: <Apps/>,
+        label: 'ButtonToggle',
+        defaultPressed: false
```

Comment:
You don't need to pass the default props to the story because you can easily see how the component is set up by default.
So you remove `size` and `defaultPressed`